### ISSUE TITLE
Include processor arch in tunnel binary directory

### ISSUE THREAD

**Issue by jason0x43** [theintern/digdug#74]
Tunnel drivers are downloaded to directories in `@theintern/digdug/selenium-standalone` based on the driver version. This allows dig dug to download download a new version of a driver binary when an existing version of the binary has already been downloaded (e.g., if a user already has IEDriverServer vX but tells dig dug to use vY, dig dug knows the existing one isn't the requested one).

However, there are cases when the processor architecture is significant as well. For example, each version of IEDriverServer comes in win32 and x64 variants. If a user has already downloaded the x64 variant, switching to the win32 would require the user to take some manual action, such as removing the existing driver or reinstalling dig dug.

In summary, driver directories should include the processor architecture as well as the version.
